sqrl-testing/sqrl-testing-integration/src/test/java/com/datasqrl/AbstractFullUseCaseTest.java

```diff
@@ -133,10 +132,8 @@ void fullUseCaseTest(UseCaseParam param) {
               .resolve(SqrlConstants.DEPLOY_DIR_NAME)
               .resolve(SqrlConstants.PLAN_DIR);
       var flinkConfig = loadInternalTestFlinkConfig(planDir, env);
-      var outputMgr = new TestOutputManager(rootDir);
-      var test =
-          new DatasqrlTest(
-              rootDir, planDir, packageJson, flinkConfig, env, outputMgr, new NoOutputFormatter());
+      var formatter = new DefaultOutputFormatter(true);
+      var test = new DatasqrlTest(rootDir, planDir, packageJson, flinkConfig, env, formatter);
```

Comment:
I'd keep `NoOutputFormatter()` here, there are specific IT cases to test the output format, and I don't like the maven formatting in our CI, which is maven as well, and that makes the whole thing more complicated to read.